src/components/Landing.js

```diff
@@ -365,21 +368,53 @@ export default class Landing extends Component {
     if (!overviewSection) {
       return false;
     }
-    let stats = [];
+    let stats = {};
     let config = overviewSection.statsConfig;
     if (config) {
       let dataSource = summary[config.dataKeySource] ? summary[config.dataKeySource][config.dataKey]: null;
       let statsSource = dataSource ? dataSource : [];
       if (config.data) {
         config.data.forEach(item => {
-          let o = statsSource;
-          if (item.keyMatch) {
+          let o = statsSource, dataSet = [], statItem = {};
```

Comment:
Minor nitpick: using several different single-character variable names in a deeply nested loop can make it difficult to understand what's going on. Longer variable names can also make find/replace easier